llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -7368,6 +7400,9 @@ SDValue RISCVTargetLowering::getAddr(NodeTy *N, SelectionDAG &DAG,
     // expands to (addi (auipc %pcrel_hi(sym)) %pcrel_lo(auipc)).
     return DAG.getNode(RISCVISD::LLA, DL, Ty, Addr);
   }
+  case CodeModel::Large: {
+    return getLargeAddr(N, DL, Ty, DAG);
```

Comment:
Can we inline this function into the switch? It doesn't look any more complicated than the other code model code.